cardano-cli/test/cli/transaction-build-raw/data/shelley-hex.address

```diff
@@ -1 +1 @@
-01c94b307274d342c0b190555c2383a1bf5afad9aa6fdbe8bb25a0d636c04dc65b23ae816a6bd6c83f99e73791577c49f2c4d88748708cbd46ea638d9a8b85cb5b
+00504f39691562f7dfd0441c15d95c10c4b4b29036cf2d96146652e2e068d094c1f3d2c8d1cb247acc00153ff1cc4512268302d4d1e5536b32c11a413bbec9818c
```

Comment:
And this will go once we merge https://github.com/input-output-hk/ouroboros-network/pull/2124